src/base/TilesRendererBase.js

```diff
@@ -302,13 +302,6 @@ export class TilesRendererBase {
 
 			}
 
-			if ( tile.content.uri ) {
-
-				// tile content uri has to be interpreted relative to the tileset.json
-				tile.content.uri = new URL( tile.content.uri, tileSetDir + '/' ).toString();
-
-			}
-
```

Comment:
What's the reason for this having to move?

Reply:
I need to keep the original content uri.
I've reverted some changes, but I had to move up the `uri` declaration.